File: StudentCredit.Application/Credits/Dtos/CreditSearchResultItemDto.cs
using StudentCredit.Data.Applications.ApplicationOne.Enums;
using StudentCredit.Data.Applications.Common.Enums;
using StudentCredit.Data.Common.Enums;
using StudentCredit.Infrastructure.Helpers.Dtos;

namespace StudentCredit.Application.Credits.Dtos
{
    public class CreditSearchResultItemDto
	{
		public int Id { get; set; }

		public DateTime ContractDate { get; set; }

		public string CreditNumber { get; set; }

		public CreditType CreditType { get; set; }

		public NomenclatureDto Bank { get; set; }

		public string StudentFullName { get; set; }

		public string Uin { get; set; }

		public string Uan { get; set; }

		public EducationType EducationType { get; set; }

		public NomenclatureDto EducationalQualification { get; set; }

		public string Institution { get; set; }

		public DateTime? BirthDate { get; set; }

		public CommitState CommitState { get; set; }

		public int? RootId { get; set; }

		public bool? PaidByApplicationFour { get; set; }

		public NomenclatureDto ApplicationType { get; set; }

		public int? ApplicationOneId { get; set; }
	}
}

File: WebApplication/src/modules/application/credit/models/credit-search-result-item.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { EducationType } from "../../application-one/enums/education-type.enum";
import { CreditType } from "../../common/enums/credit-type.enum";
import { CommitState } from "../../common/enums/commit-state.enum";

export class CreditSearchResultItemDto {
  id: number;
  contractDate: Date;
  creditNumber: string;
  creditType: CreditType;
  bank: NomenclatureDto;
  studentFullName: string;
  uin: string;
  uan: string;
  educationType: EducationType;
  educationalQualification: NomenclatureDto;
  institution: string;
  birthDate: Date;
  commitState: CommitState;
  rootId: number;
  paidByApplicationFour: boolean;
  applicationType: NomenclatureDto;
  applicationOneId: number;
}
